cmake/std/atdm/van1-tx2/environment.sh

```diff
@@ -84,6 +84,27 @@ elif [[ "$ATDM_CONFIG_COMPILER" == "ARM-20.1_OPENMPI-4.0.3" ]]; then
     unset OMP_PROC_BIND
   fi
 
+  # We'll use TPL_ROOT for consistency across ATDM environments
+  export MPI_ROOT=${MPI_DIR}
+  export BLAS_ROOT=${ARMPL_DIR}
+  export HDF5_ROOT=${HDF5_DIR}
+  export NETCDF_ROOT=${NETCDF_DIR}
+  export PNETCDF_ROOT=${PNETCDF_DIR}
+  export ZLIB_ROOT=${ZLIB_DIR}
+  export CGNS_ROOT=${CGNS_DIR}
+  export METIS_ROOT=${METIS_DIR}
+  export PARMETIS_ROOT=${PARMETIS_DIR}
+  export SUPERLUDIST_ROOT=${SUPERLU_DIST_DIR}
+  export BINUTILS_ROOT=${BINUTILS_DIR}
+elif [[ "$ATDM_CONFIG_COMPILER" == "ARM-20.1_OPENMPI-4.0.5" ]]; then
+  module load sparc-dev/arm-20.1_openmpi-4.0.5
+  module unload yaml-cpp
+
+  if [ "$ATDM_CONFIG_NODE_TYPE" == "OPENMP" ] ; then
+    unset OMP_PLACES
+    unset OMP_PROC_BIND
```

Comment:
I think I remember @rppawlo saying this `OMP_PROC_BIND` is something we want to leave up to the user rather than set (or unset) across the board.  Might be wrong, though&mdash;memory is foggy.

Reply:
I'd be inclined to agree with you, but it should only affect _runtime_ behavior, not build-time, correct?

Reply:
I think that's right.

Reply:
FYI: Experiments with Trilinos showed that unsetting these improved the performance of running the Trilinos test suite.

Reply:
Understood, but I think it was unsetting these improved testing, where you're firing off a bunch of small things at once, but degraded performance of larger runs where you're launching one big thing.  Hence the desire to leave it up to the user.

Reply:
> Hence the desire to leave it up to the user.

Right, but out of the box, this should work for the Trilinos test suite or someone will need to deal with a bunch of timing out Trilinos tests.